### PR TITLE
Avoid notice in gcontact.php

### PR DESCRIPTION
**Pull request by annando**
Fixes `PHP Notice:  Undefined variable: fields`

### CODE REVIEW

File: src/Model/GContact.php [hunk 1]
```diff
@@ -1343,14 +1343,18 @@ public static function discoverFollowers(string $url)
 			foreach ($contacts as $contact) {
 				$gcontact = DBA::selectFirst('gcontact', ['id'], ['nurl' => Strings::normaliseLink(($contact))]);
 				if (DBA::isResult($gcontact)) {
+					$field = [];
```
Comment (MrPetovan) [inline src/Model/GContact.php]:
```suggestion
					$fields = [];
```

Comment (annando) [reply src/Model/GContact.php]: Möööp

Comment (MrPetovan) [reply src/Model/GContact.php]: That's a new one. I knew "Örgs" but not "Möp".

Comment (annando) [reply src/Model/GContact.php]: I'm a bag of surprise for new words :-)

**Pull request merged**